tests/devsim_test2_gold.json

```diff
@@ -58,61 +32,61 @@
     }
   ],
   "VkPhysicalDeviceFeatures": {
-    "alphaToOne": 118,
-    "depthBiasClamp": 113,
-    "depthBounds": 115,
-    "depthClamp": 112,
-    "drawIndirectFirstInstance": 111,
-    "dualSrcBlend": 108,
-    "fillModeNonSolid": 114,
-    "fragmentStoresAndAtomics": 127,
-    "fullDrawIndexUint32": 102,
-    "geometryShader": 105,
-    "imageCubeArray": 103,
-    "independentBlend": 104,
-    "inheritedQueries": 155,
-    "largePoints": 117,
-    "logicOp": 109,
-    "multiDrawIndirect": 110,
-    "multiViewport": 119,
-    "occlusionQueryPrecise": 124,
-    "pipelineStatisticsQuery": 125,
-    "robustBufferAccess": 101,
-    "sampleRateShading": 107,
-    "samplerAnisotropy": 120,
-    "shaderClipDistance": 138,
-    "shaderCullDistance": 139,
-    "shaderFloat64": 140,
-    "shaderImageGatherExtended": 129,
-    "shaderInt16": 142,
-    "shaderInt64": 141,
-    "shaderResourceMinLod": 144,
-    "shaderResourceResidency": 143,
-    "shaderSampledImageArrayDynamicIndexing": 135,
-    "shaderStorageBufferArrayDynamicIndexing": 136,
-    "shaderStorageImageArrayDynamicIndexing": 137,
-    "shaderStorageImageExtendedFormats": 130,
-    "shaderStorageImageMultisample": 131,
-    "shaderStorageImageReadWithoutFormat": 132,
-    "shaderStorageImageWriteWithoutFormat": 133,
-    "shaderTessellationAndGeometryPointSize": 128,
-    "shaderUniformBufferArrayDynamicIndexing": 134,
-    "sparseBinding": 145,
-    "sparseResidency16Samples": 152,
-    "sparseResidency2Samples": 149,
-    "sparseResidency4Samples": 150,
-    "sparseResidency8Samples": 151,
-    "sparseResidencyAliased": 153,
-    "sparseResidencyBuffer": 146,
-    "sparseResidencyImage2D": 147,
-    "sparseResidencyImage3D": 148,
-    "tessellationShader": 106,
-    "textureCompressionASTC_LDR": 122,
-    "textureCompressionBC": 123,
-    "textureCompressionETC2": 121,
-    "variableMultisampleRate": 154,
-    "vertexPipelineStoresAndAtomics": 126,
-    "wideLines": 116
+    "alphaToOne": 1,
```

Comment:
Were the old values IDs of some kind?  I don't understand why these were changed.

Reply:
These values are `VkBool32` which can technically be any 32-bit integer.

The original test made these values weird, ridiculous numbers to show that devsim could simulate a driver with values that didn't make any sense.

Due to updates in `vulkaninfo`, these values are forced to `0` or `1` in output, so in order for the tests to pass now, the gold files had to be updated to the new `vulkaninfo` output.